src/components/TopBar.tsx

```diff
@@ -135,7 +135,15 @@ const DropDownSection = (props: DropDownSectionProps): ReactNode => {
   );
 };
 
-const TopBar = ({ showMenu = true, title, href, children }) => {
+interface TopBarProps {
```

Comment:
This could extend `PropsWithChildren` like we do elsewhere.

Reply:
I was avoiding that, because I thought there was some sort of reason we _didn't_ want to do that 😆